numpy/core/fromnumeric.pyi

```diff
@@ -392,7 +393,7 @@ def clip(
     order: _OrderKACF = ...,
     subok: bool = ...,
     signature: str | tuple[None | str, ...] = ...,
-    extobj: list[Any] = ...,
```

Comment:
I don't believe the removal of `extobj` is correct here; https://github.com/numpy/numpy/pull/23922 is reserved for the 2.0 release, no?

Reply:
Beats me :) But if you think it needs to stay, I'll revise that part.

Reply:
See #24617.